### PR TITLE
chore: changelog 0.20.0

### PR DESCRIPTION
**Pull request by angelozerr**
chore: changelog 0.20.0

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,17 @@
 # Quarkus extension for MicroProfile Language Server Changelog
 
+## [0.20.0](https://github.com/redhat-developer/quarkus-ls/milestone/23?closed=1) (October 17, 2024)
+
+### Enhancements
+
+- feat: Available Ollama models support for quarkus.langchain4j.ollama.chat-model.model-id property value. See [#985](https://github.com/redhat-developer/quarkus-ls/pull/985).
```
Comment (fbricon) [inline CHANGELOG.md:7]:
```
completion on available Ollama models support for `quarkus.langchain4j.ollama.chat-model.model-id` ...
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,17 @@
 # Quarkus extension for MicroProfile Language Server Changelog
 
+## [0.20.0](https://github.com/redhat-developer/quarkus-ls/milestone/23?closed=1) (October 17, 2024)
```
Comment (rgrunber) [inline CHANGELOG.md:3]: I would get rid of the 'feat' / 'fix' prefixes. We know what they are based on the headings.

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,17 @@
...
+
+### Enhancements
+
+- feat: Available Ollama models support for quarkus.langchain4j.ollama.chat-model.model-id property value. See [#985](https://github.com/redhat-developer/quarkus-ls/pull/985).
+
+### Bug Fixes
+
+- fix: add languageId filter for options similar to microprofile. See [#999](https://github.com/redhat-developer/quarkus-ls/pull/999).
+- fix: Message: Request qute/java/documentLink failed with message: Cannot invoke "org.eclipse.jdt.core.dom.ITypeBinding.getQualifiedName()". See [#996](https://github.com/redhat-developer/quarkus-ls/pull/996).
+- NPE with {# and Qute inlay hint. See [#991](https://github.com/redhat-developer/quarkus-ls/issues/991).
```
[the remark this answers is not in the document]
Comment (angelozerr) [reply CHANGELOG.md]: fixed

File: CHANGELOG.md (hunk 2 above)
Comment (angelozerr) [reply CHANGELOG.md]: fixed

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,5 +1,23 @@
...
+## [0.20.0](https://github.com/redhat-developer/quarkus-ls/milestone/23?closed=1) (October 22, 2024)
+
+### Enhancements
+
+- Completion on available Ollama models support for `quarkus.langchain4j.ollama.chat-model.model-id` property value. See [#985](https://github.com/redhat-developer/quarkus-ls/pull/985).
+- `quarkus.rest.path` support. See [#1003](https://github.com/redhat-developer/quarkus-ls/pull/1003).
+
+### Bug Fixes
+
+- Fix false-positive error when string parameter contains a comma. See [#1011](https://github.com/redhat-developer/quarkus-ls/issues/1011).
```
[the remark this answers is not in the document]
Comment (angelozerr) [reply CHANGELOG.md:12]: No it is a bug that I have introduced with Ollama support which is only available for this version.

**Review by datho7561 — APPROVED** [review]
Looks good to me. Thanks, Angelo!

**Pull request merged**